### ISSUE TITLE
Add accessibilityException property to describe undue/disproportionate burden in accessibility metadata

### ISSUE THREAD

**Issue by monikagithub** [schemaorg/schemaorg#4472]
I propose a new property accessibilityException (or alternatively accessibilityLimitationReason) for use on CreativeWork and related types. This property would describe reasons why accessibility features could not be provided, such as undue burden, disproportionate burden, or technical infeasibility.

While the accessibilitySummary property allows for a general text summary, it is not semantically explicit, and not machine-readable for regulatory or audit purposes.

ONIX Code 196 already has properties to add this kind of metadata.

For Journal Articles to comply with ADA Title II legislation, we are using schema.org accessibility metadata properties and it would be really helpful to have this property.

Kind Regards

**Comment by chaals** [comment]
Do you propose that the content would be machine-readable, or is it just 
the ability to deterministically find the explanation for why something has 
problems?

Absent any real ability to process it, I lean towards thinking that it's 
the sort of information that *should* be in the accessibilitySummary. In 
part this is because I am wary of encouraging people to set up a default 
value for it instead of doing anything about solving problems, but in part 
I think that's one of the points of the existing summary property - other 
than for auditing purposes, I am not sure who the information serves, and 
when trying to find an accessible resource I suspect there are few people 
who care about specifically why some given resource didn't get built 
properly.

**Comment by monikagithub** [comment]
Thanks for your reply.
My first inclination is also towards acessibilitySummary. There are a few issues we are facing if we add this in Accessibility Summary. There are a few other things like link to accessibility statement, WCAG adherence, etc, that we may need to show in accessibilitySuammary as there are no specific properties available for those. The problem is when displaying these properties as human readable text to the customers. As per the display guidelines, some of these properties fit in one section and some in the other. I don't suppose we can display the text split out from the accessibilitySummary (it would be clunky even if we could). EAA and ADA allow for disproportionateBurden/Fundamental Alteration etc, but there has to be a proper reason and documentation for it. This would different from "Unknown" Accessibility because if there is an Undue Burden (for example) and one or more customers request for that resource to be made accessible (we may reconsider and reassess the impact of the cost or resources of making that accessible). It is more informative to the customer of why the resource is not Accessible and if there are any alternates available. Other most important factor is the Audit. We would need to easily identify resources marked as disproportionate/undue burden and it would be really helpful to have a separate property for it. I understand it may not be possible to add a new property for just one request. Not sure if anyone else has made similar requests.

Kind Regards
Monika

**Comment by gautierchomel** [comment]
If it helps, there is an [EPUB Accessibility exemption property](https://www.w3.org/TR/epub-a11y-exemption/) and a guidance for accessibility summary authoring in the [Expressing Accessibility Metadata in the EPUB Package Document](https://w3c.github.io/publ-a11y/epub-a11y-meta-guide/1.0/draft/index.html#accessibilitySummary).
